src/components/Quiz.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class Quiz extends Component {
+  constructor() {
+    super();
+    this.state = {
+      quizzes: [],
+      questions: [],
+    };
+  }
+
+  componentDidMount() {
+    axios.get('/quizzes', {
+    })
+    .then((response) => {
+      this.setState({
+        quizzes: response.data.quizzes,
+        questions: response.data.quizzes[0].questions,
+      });
+    })
+    .catch(function () {
+      console.log('request failed');
+    });
+   }
+
+  renderQuizTitleToPage(state) {
+    return state.map(q => <article className='quiz-title' key={q.id}>
+                          <h1>{q.title}</h1>
+                        </article>)
+  }
+
+  renderQuestionsToPage(state) {
+    return state.map(q => <article className='question-answers' key={q.id}>
```

Comment:
We might want to break up this block of HTML we're returning by creating some additional components. If we add a `Question` component and an `Answer` component, we could render something like the following:

```javascript
questions.map((question, index) =>
    <Question title={question.title} key={question.id}/>
    (question.answers).map(answer =>
        <Answer title={answer.title} />
    )
```

(That is not going to be the exact syntax, and I'm sure it will break in all sorts of ways, but that's what we should be going for here.)